opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -117,8 +117,7 @@ class EclMaterialLawManager
                       const std::vector<int>& compressedToCartesianElemIdx)
     {
         // get the number of saturation regions and the number of cells in the deck
-        int ntsFun = deck.getKeyword("TABDIMS").getRecord(0).getItem("NTSFUN").get<int>(0);
-        unsigned numSatRegions = static_cast<unsigned>(ntsFun);
+        const size_t numSatRegions = eclState.runspec().tabdims().getNumSatTables();
```

Comment:
to fix the travis complaint, this should be `eclState.runspec().getTabdims()`?

@joakim-hove: is there a reason why the Runspec object is not accessed via `eclState.getRunspec()`?